lib/orbit/import/import_helpers.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Orbit.Import.ImportHelpers do
+  @moduledoc """
+  Helpers for importing CSVs from HASTUS
+  """
+
+  @spec parse_csv(Enumerable.t(), [atom()] | boolean(), keyword()) :: Enumerable.t()
+  def parse_csv(file_lines, headers, options \\ []) do
+    file_lines
+    |> Stream.drop(
+      if headers == true do
+        0
+      else
+        1
+      end
+    )
+    |> Stream.map(fn line ->
+      if String.ends_with?(line, "\n") do
+        line
+      else
+        "#{line}\n"
+      end
+    end)
+    |> CSV.decode!(
+      options ++ [headers: headers, field_transform: &String.trim/1, validate_row_length: true]
+    )
+  end
+
+  @spec to_csv(Enumerable.t(), [atom()]) :: [String.t()]
+  def to_csv(data, headers) do
```

Comment:
YAGNI, and also this wouldn't be used for importing. Should probably be removed.